README.md

```diff
@@ -92,15 +93,12 @@ py.test -v tests
 ## Releasing
 
 1. Increment `__version__` in `setup.py`
-1. Summarize release changes in `NEWS.md`
-1. Commit these changes, then tag the release
-  ```bash
+2. Summarize release changes in `NEWS.md`
```

Comment:
Technically, in Markdown [numbers are ignored](https://www.markdownguide.org/basic-syntax#ordered-lists), but this change is fine.

Reply:
I had no idea!